README.md

```diff
@@ -1,7 +1,7 @@
 #Pome
 Pome stands for **Po**stgres **Me**trics.
 Pome is a PostgreSQL Metrics Dashboard to keep track of the health of your database.
-This project is at a very early stage and there are a lot of missing features,
+This project is at a very early stage and there are a lot of missing features
```

Comment:
That's not always true. See http://www.grammarly.com/handbook/punctuation/comma/23/comma-before-but

However, the sentence is readable either way so doesn't make much of a difference.